src/test/resources/dataset/contacts.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dataset>
+    <contact
+        id="1" username="username12345" password="$2a$10$3Ds5qfmI2JcqluYe5cBp1Onak2GL5douZSxGgViQUx9FsLuQT2FkC" is_active="true" is_private="true" email="user@gmail.com" role="USER"
```

Comment:
Could you place each field of entity in a new line so it would be much more readable?